### ISSUE TITLE
If `getValue` is declared as an extension method ktlint thinks its unused

### ISSUE THREAD

**Issue by JLLeitschuh** [shyiko/ktlint#54]
Example:

```kotlin
import com.natpryce.konfig.PropertyGroup
import com.natpryce.konfig.getValue
import com.natpryce.konfig.stringType

interface EventConfiguration {
    val url: String

    companion object {
        private object rabbitmq : PropertyGroup() {
            val url by stringType // This is where getValue is used.
        }
}
```

This is how `getValue` is defined:
```kotlin
operator fun <G : PropertyGroup, T> ((PropertyLocation, String) -> T).getValue(group: G, property: KProperty<*>) =
        group.key(property.name, this)

operator fun <SCOPE, T> ((PropertyLocation, String) -> T).getValue(scope: SCOPE?, property: KProperty<*>) =
        Key(property.name, this)
```

Kt-lint thinks that `import com.natpryce.konfig.getValue` is an unused import so you have to disable the check using `// ktlint-disable`.

**Comment by shyiko** [comment]
@JLLeitschuh Thank you. Fixed in [0.8.2](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#082---2017-06-06) (Maven Central sync might take up to 25min).

**Comment by JLLeitschuh** [comment]
While you are at it you should fix it for `setValue`
